RELEASE_NOTES.md

```diff
@@ -32,16 +43,26 @@ English version below
 
 - Removal of script ``ffe.bat`` (2.7.0)
 - Added the lottery (2.7.0)
+- Accept no players in screen families (2.7.1)
 
 ## Web interface
 
 - Optimized the load time of the events page (2.7.0)
 - Instant notifications on Players and Pairings tabs for new checkins or results from user screens (2.7.0)
+- Forbid access to arbiters' pages for unauthorized clients (2.7.1)
 
 ### Tournaments management
 
 - Added PGN and TRF 16 exports (2.7.0)
 
+# Players management
+
+- Fixed and added columns to the players ODS export (2.7.1)
+
+# Pairings management
+
+- Reduced reaction times when entering results (2.7.1)
```

Comment:
```suggestion
- Improve perceived performance times when entering results (2.7.1)
```

Reply:
Actually I feel that "Reduced reaction times when entering results" is pretty exact.  The reaction time is no longer held back by the server.  A side effect is that the perceived performance is improved, but the reduced reaction time is real.